Dockerfile

```diff
@@ -9,12 +9,24 @@ RUN \
     lsblk \
     dmidecode \
     util-linux \
-    lm-sensors \
-    speedtest-cli &&\
+    lm-sensors &&\
   apk --no-cache add \
     raspberrypi \
     || true
 
+ARG TARGETPLATFORM
+RUN \
+  if [ "$TARGETPLATFORM" = "linux/amd64" ] ; \
+    then wget -qO- https://install.speedtest.net/app/cli/ookla-speedtest-1.1.1-linux-x86_64.tgz \
+    | tar xmoz -C /bin speedtest ; \
+  elif [ "$TARGETPLATFORM" = "linux/arm64" ] ; \
+    then wget -qO- https://install.speedtest.net/app/cli/ookla-speedtest-1.1.1-linux-aarch64.tgz \
+    | tar xmoz -C /bin speedtest ; \
+  elif [ "$TARGETPLATFORM" = "linux/arm/v7" ] ; \
+    then wget -qO- https://install.speedtest.net/app/cli/ookla-speedtest-1.1.1-linux-armel.tgz \
```

Comment:
According to [this table](https://github.com/BretFisher/multi-platform-docker-build#the-problem-with-downloading-binaries-in-dockerfiles) the ending "armel" is actually for v6 and "armhf" for v7.

Reply:
Oh okay, I wasn't sure. Will fix that 👍🏻